src/customConditions.ts

```diff
@@ -0,0 +1,13 @@
+const customConditions = {
+    greaterThan: (value, threshold) => value > threshold,
+    lessThan: (value, threshold) => value < threshold,
```

Comment:
Lets support shorthand as well. like gt, lt, eq, etc. objective is that, user may not have to write long in yml